designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -245,28 +246,23 @@ class ManagementResources(val processAuthorizer: AuthorizeProcess,
             (post & processId(processName) & entity(as[CustomActionRequest])) { (process, req) =>
               val params = req.params.getOrElse(Map.empty)
               complete {
-                customActionInvokerService.invokeCustomAction(req.actionName, process, params)
-                  .flatMap {
-                    case res@Right(_) =>
-                      toHttpResponse(CustomActionResponse(res))(StatusCodes.OK)
```

Comment:
Am I right that you changed an API contract and now instead of JSON with `isSuccess` and `msg` fields we return an empty response?

Reply:
I've reverted CustomActionResponse, but I think we should change it: `msg` seems to be relevant, but attribute `isSuccess` is redundant with `StatusCodes.OK`.

Reply:
I'm not a great fan of the current API, but I assume that when we change the interface, we have to make some changes on the FE side as well. Don't we? But also - let's postpone these changes because after rewriting hardcoded actions to custom, this interface may change.